src/core/engines/Cesium/Feature/Tileset/index.tsx

```diff
@@ -43,17 +207,25 @@ export default function Tileset({
 
   const tilesetUrl = useMemo(() => {
     return sourceType === "osm" && isVisible
-      ? IonResource.fromAssetId(96188) // https://github.com/CesiumGS/cesium/blob/1.69/Source/Scene/createOsmBuildings.js#L50
+      ? IonResource.fromAssetId(1240402) // https://github.com/CesiumGS/cesium/blob/1.69/Source/Scene/createOsmBuildings.js#L50
```

Comment:
```suggestion
      ? IonResource.fromAssetId(96188) // https://github.com/CesiumGS/cesium/blob/main/packages/engine/Source/Scene/createOsmBuildings.js#L53
```

Reply:
This is fixed in [Update src/core/engines/Cesium/Feature/Tileset/index.tsx](https://github.com/reearth/reearth-web/pull/377/commits/bd66b8c863dc52387db015e7a6078dde4e86181f)